<launch>
    

    <node pkg="rosbag" type="play" name="play" args="/home/libing/data/rosbag/kitti_stereo/kitti_07.bag" />

    <node pkg="mono_vo" type="mono_vo" name="mono_vo" output="screen" >
        <remap from="/left_image" to="/kitti_stereo/left/image_rect"/>
        <remap from="/right_image" to="/kitti_stereo/right/image_rect"/>
        <remap from="/camera_info" to="/kitti_stereo/right/camera_info"/>
    </node>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find mono_vo)/rviz_cfg/mono_vo.rviz"/>
</launch>